doc/conf.py

```diff
@@ -12,9 +12,8 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys
-import os
-sys.path.insert(0, os.path.abspath('..'))
+from libnacl import __version__ as version
```

Comment:
Actually if you're using setuptools (see other pull request) you can do
"python setup.py build_sphinx" from the project's root which will create
the documentation based on the local version and not the installed one.

-gabi

On Sun, Dec 20, 2015 at 10:32 AM, Pedro Algarvio <notifications@github.com>
wrote:

> In doc/conf.py
> <https://github.com/saltstack/libnacl/pull/69#discussion_r48101031>:
>
> > @@ -12,9 +12,8 @@
> >  # All configuration values have a default; values that are commented out
> >  # serve to show the default.
> >
> > -import sys
> > -import os
> > -sys.path.insert(0, os.path.abspath('..'))
> > +from libnacl import __version__ as version
>
> This requires libnacl to be installed to generate the documentation. The
> sys.path.insert allows the docs to be built without being installed...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/saltstack/libnacl/pull/69/files#r48101031>.
>